traffic_ops/app/lib/UI/DeliveryService.pm

```diff
@@ -78,7 +78,8 @@ sub get_cdn_domain {
 			distinct => 1
 		}
 	)->get_column('value')->single();
-	return $cdn_domain;
+	# Always return a lowercase FQDN.
+	return lc($cdn_domain);
```

Comment:
I did see those methods but from what I could tell they were not related to the paths that the DNSSEC stuff follows. I wanted to be surgical in the fix, rather than fixing every instance of things that might need to be lowercased.